### PR TITLE
Update gathering conditions

### PR DESCRIPTION
**Pull request by tremes**

### CODE REVIEW

**Comment by tremes** [comment]
Please check @Sergey1011010, @dpensi

File: conditions/APIRemovedInNextEUSReleaseInUse.json [hunk 1]
```diff
@@ -0,0 +1,16 @@
+{
+    "conditions": [
+      {
+        "type": "alert_is_firing",
+        "params": {
```
Comment (Serhii1011010) [inline conditions/APIRemovedInNextEUSReleaseInUse.json]: it should be "alert"

Comment (tremes) [reply conditions/APIRemovedInNextEUSReleaseInUse.json]: Thanks. Will update

**Comment by Serhii1011010** [comment]
here's a little fix for the validation script https://github.com/RedHatInsights/insights-operator-gathering-conditions/pull/7

**Pull request merged**